<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmwareSettingsDialog</class>
 <widget class="QDialog" name="FirmwareSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Firmware settings - melonDS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="lblInstanceNum">
     <property name="text">
      <string>Configuring settings for instance X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="overrideFirmwareBox">
        <property name="text">
         <string>Override settings from external firmware</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpUserSettings">
     <property name="title">
      <string>User settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbxBirthdayDay"/>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cbxBirthdayMonth"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="usernameEdit">
        <property name="text">
         <string>melonDS</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Message:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="colorsEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Birthday:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="messageEdit"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="languageBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpWifiSettings">
     <property name="title">
      <string>Network settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>MAC address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtMAC">
        <property name="maxLength">
         <number>17</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>(leave empty to use default MAC)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>overrideFirmwareBox</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>languageBox</tabstop>
  <tabstop>cbxBirthdayDay</tabstop>
  <tabstop>cbxBirthdayMonth</tabstop>
  <tabstop>colorsEdit</tabstop>
  <tabstop>messageEdit</tabstop>
  <tabstop>txtMAC</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>FirmwareSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>FirmwareSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
